nbgrader/exchange/default/fetch_feedback.py

```diff
@@ -2,11 +2,13 @@
 import shutil
 import glob
 
-from .exchange import Exchange
-from ..utils import check_mode, notebook_hash, make_unique_key, get_username
+from nbgrader.exchange.abc import ExchangeFetchFeedback as ABCExchangeFetchFeedback
+from nbgrader.exchange.default import Exchange
```

Comment:
Why have all the relative imports been changed? They should work as was, no?

Reply:
It was a long time since I wrote this, so can't quite remember, but I assume it was just a side effect of changing the structure of the package. It shouldn't matter either way.